typescript/phoenix-sdk/src/utils/market.ts

```diff
@@ -628,3 +633,299 @@ export function getMarketExpectedOutAmount({
 
   return expectedUnitsReceived;
 }
+
```

Comment:
I feel like this stuff belongs in its own file. Feels pretty uniquely decoupled from `market`